### ISSUE TITLE
Feature Request: Provide as an installable package

### ISSUE THREAD

**Issue by amharris** [AndrewJBateman/ionic-select-searchbar#13]
I've just come across this project, as I'm looking for a plugin that effectively offers the `IonSearchbar` interface but with the underlying functionality of acting as `IonSelect` when selecting a result and then showing it afterwards.  It's a wonderful piece of work but, unless I'm missing something, this doesn't appear to provide a package that can be installed (via NPM/Yarn) for use in other projects.

It feels native to Ionic 5 but being usable for those with their own projects and not a demo in itself would probably get this project a lot more traction and interest for those wanting to work on React and Vue support, ([per my other Feature Request](https://github.com/AndrewJBateman/ionic-select-searchbar/issues/12)).

Thanks (and great work!) 😃

**Comment by AndrewJBateman** [comment]
I didn't create the IonSelect, I just used it:

**Comment by AndrewJBateman** [comment]
I didn't create the IonSelect, I just used it:

**Comment by amharris** [comment]
Hi @AndrewJBateman 

Thanks for getting back to me, despite the time it's been.

To clarify, I understand that you didn't create the core component `IonSelect` but leveraged it as a part of your component.
What I was asking is if you'd be able to provide yours as an installable package itself, on NPM, it'd be easily reusable across real, production-ready applications without rewriting as much.

Edit: My apologies—early mornings!
I _think_ I see what you mean.  That you were just providing an example of using [Ionic Selectable](https://www.npmjs.com/package/ionic-selectable)?  If so, thanks for clarifying.  When I initially looked at your repository, given how fleshed out the `README` is and that Ionic Selectable states 'Vanilla JS', I took it that you had put together a wrapper of sorts around it.
Not to worry.  Though a React and Vue example [per my other request](https://github.com/AndrewJBateman/ionic-select-searchbar/issues/12) would still be cool, (not that you're obliged—at the time I was still learning React and trying to recreate what you had done, but it wasn't quite clicking.  I took a different approach in the end, but I'd still be interested in seeing how _you'd__ approach this for React/Vue).

Anyway, thanks again for getting back to me.
